templates/vue/src/components/Lightbox/media/MultiContentMedia/AccordionRows.vue

```diff
@@ -184,9 +181,9 @@ export default {
     },
     rowBackground() {
       if (this.isMultiContentContext) {
-        let rgb = 40
-        let colorOffset = this.level * 10
-        rgb = colorOffset > rgb ? 0 : rgb - colorOffset
+        let rgb = 187
+        // let colorOffset = this.level * 10
+        // rgb = colorOffset > rgb ? 0 : rgb - colorOffset

```

Comment:
This dynamic coloring should not be removed as I believe this is what makes the nested accordions visible as they each will have a different shade on top of their parent accordion.

Reply:
reversed